check-requirements.in

```diff
@@ -1,7 +1,16 @@
 bandit
+darglint
 flake8
+flake8-annotations-complexity
+flake8-bugbear
```

Comment:
Marks general questionable code known to cause bugs (like `raise NotImplemented`).

Reply:
Bugbear doesn't seem to have a rule for `raise NotImplemented`. It does for `raise False` though.

Reply:
I do absolutely love this rule though:

> *B950*: Line too long. This is a pragmatic equivalent of pycodestyle's E501: it considers "max-line-length" but only triggers when the value has been exceeded by more than 10%. You will no longer be forced to reformat code due to the closing parenthesis being one character too far to satisfy the linter. At the same time, if you do significantly violate the line length, you will receive a message that states what the actual limit is. This is inspired by Raymond Hettinger's "Beyond PEP 8" talk and highway patrol not stopping you if you drive < 5mph too fast. Disable E501 to avoid duplicate warnings.

Reply:
This one seems to be in the opinionated set of rules right? It's not turned on by default

Reply:
Yes that's right

Reply:
True, maybe it'd be good idea to toy with some of the opinionated ones too :thinking: 

I figured bugbear's B016 should catch `raise NotImplemented` as raising a literal, but it's picked up as F109, I lost which plugin trips it :')